dev/io.openliberty.netty.internal.impl/test/io/openliberty/netty/internal/impl/NettyFrameworkImplTest.java

```diff
@@ -44,7 +44,8 @@
  */
 public class NettyFrameworkImplTest {
 
-    private static SharedOutputManager outputMgr = SharedOutputManager.getInstance();
+    private static SharedOutputManager outputMgr = SharedOutputManager.getInstance()
+            .trace("io.netty*=all:io.openliberty.netty*=all");
```

Comment:
Probably worth keeping the trace string as a constant?

Reply:
now NettyConstants.NETTY_TRACE_STRING